examples/meteor/v1beta1/meteor-controller.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Given that v1beta1 is going away soon, I'd rather not clutter the depot with them. Please remove them.